src/Plugin/os2web/NemloginAuthProvider/OpenIDConnect.php

```diff
@@ -97,6 +97,11 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function isInitialized() {
+    $configuration = $configuration = $this->getConfiguration();
+    if (isset($configuration['nemlogin_openid_connect_discovery_url']) && !empty($configuration['nemlogin_openid_connect_discovery_url'])) {
```

Comment:
`!empty($configuration['nemlogin_openid_connect_discovery_url'])` (i.e. no `isset`) should be enough here.

Reply:
agree, changed